package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@adia-technology/policies-consent-widget",
   "version": "1.1.2",
-	"description": "Angular module to manage terms, policies and Smartlook integration.",
-	"main": "dist/policies-consent-widget/public-api.d.ts",
+  "description": "Angular module to manage terms, policies and Smartlook integration.",
+  "main": "dist/policies-consent-widget/public-api.d.ts",
```

Comment:
`main` must point to the JS entry point, not TS definition file.

Reply:
fixed